index.ge.ts

```diff
@@ -0,0 +1,17 @@
+*2021-03-23*
+
+Richard M. Stallman ცნობილია RMS- ის სახელით. რომელიც რა ხანია საფრთხეს წარმოადგენს თავისუფალი პროგრამული უზრუნველყოფის საზოგადოებაში. შემდეგ სერიოზულ ბრალდებებთან ერთად: როგორიცაა ქალთმოძულეობა, კალისტურუბა თუ ტრანსფობია. ამ სახის რწმენას ადგილი ნამდვილად არ აქვს პროგრამულ უზრუნველყოფაში, ციფრულ უფლებებსა და ზოგადად თავისუფალი ტექნოლოგიების საზოგადოებაში. თავისუფალი ფონდის საბჭოს აღდგენის შემდეგ, ჩვენ ვითხოვთ, FSF– ის მთელი საბჭოს გადადგომას და აგრეთვე RMS ხელმძღვანელის მოხსნას დაკისრებული თანამდებობიდან.
+ 
+ჩვენი ხელმომწერებით, გვინდა მივაღწიოთ ციფრული ავტონომიის საჭიროებას. რისი მთავარი როლიც მომხმარებლის თავისუფლება და ადამიანის უფლებების დაცვაა. იმისთვის რათა შეასრულდეს დაპირება პროგრამული უზრუნველყოფის თავისუფლებისთვის საზოგადოებაში უნდა მოხდეს ძირეული ცვლილებები. ჩვენ გვწამს აწმყოსა და მომავალში, იმ ტექნოლოგიების სადაც ხალხი ვითარდება და არა ვიწროვდება. ჩვენ ვიცით, რომ ეს შესაძლებელია მხოლოდ იმ სამყაროში, მხოლოდ იმ სივრცეში სადაც ტექნოლოგია აგებულია ჩვენი უფლებების პატივისცემისათვის. მიუხედავად იმისა, რომ ამ იდეებს გარკვეულწილად პოპულარიზაციას უწევდა Richard M. Stallman, ჩვენ მაინც არ ვეთანხმებით მის ქმედებებს და მოსაზრებებს. 
+ 
+საკმაოდ დიდხანს ვიყავით ტოლერანტულები RMS- ის იდეების და ქცევების. ვერ დავუშვებთ რომ თუნდაც ერთმა ადამიანმა ხელი შეუშალოს და დაანგრიოს ჩვენი საქმიანობა. ჩვენს საზაოგადოებს ადგილი არ აქვს ადამიანებისთვის როგორიცაა Richard M. Stallman და ჩვენ ვეღარ გავაგრძლებთ მის დანდობასა რომ მივანიჭოთ ლიდერის როლი და  ავიტანოთ მისი ჩვეულებები. მისი იდეოლიგია არის შეურაცმყოფელი და საშიში
+
+**ჩვენი მოთხოვნაა მოიხსნას Free Software Foundation ფონდის დირექტორთა საბჭო.** ეს ის ხალხია, ვინც წლების განმავლობაში აძლიერებს და აძლიერებს RMS- ს. დროა RMS გამოვიდეს უფასო პროგრამიდან, ტექნიკური ეთიკიდან, ციფრული უფლებებიდან და ტექნიკური საზოგადოებებიდან, რადგან ის ვერ უზრუნველყოფს ჩვენთვის საჭირო ხელმძღვანელობას. **ჩვენ ასევე ვთხოვთ, რომ Richard M. Stallman მოხსნას ყველა ხელმძღვანელი თანამდებობიდან, მათ შორის GNU პროექტიდან.**
+ 
+ჩვენ მოვუწოდებთ მათ, რომ შეაჩერონ Free Software Foundation მხარდაჭერა. უარი თქვით FSF და RMS პროექტებში მონაწილეობაზე. არ ისაუბროთ FSF– ის ღონისძიებებზე, არ დაესწროთ ღონისძიებებს, რომელსაც მასპინძლობს RMS. ჩვენ ვთხოვთ უფასო პროგრამული უზრუნველყოფის პროექტების ავტორებს, მიიღონ თავიანთი პროექტების მიმართ პოზიტიური დამოკიდებულება.
```

Comment:
Would appreciate if we had another Georgian speaker review!

`ჩვენ ვთხოვთ უფასო პროგრამული უზრუნველყოფის პროექტების ავტორებს, მიიღონ თავიანთი პროექტების მიმართ პოზიტიური დამოკიდებულება.` Google translates to `We ask the authors of free software projects to get a positive attitude towards their projects.` which doesn't seem to match `While doing these things, tell these communities and the FSF why.`

Reply:
Google translate is pretty terrible in Georgian, since it's such a small country they do not invest much in it. I'll try to find another Georgian speaker to review.

Reply:
Thank you! It's definitely a problem for certain languages. I had to find native speakers to help with e.g. Arabic as well.